x-pack/platform/test/onechat_api_integration/apis/esql_tools.ts

```diff
@@ -125,6 +126,71 @@ export default function ({ getService }: FtrProviderContext) {
       });
     });
 
+    describe('POST /api/chat/tools/_execute', () => {
+      const testIndex = 'test-onechat-index';
+
+      before(async () => {
+        await es.indices.create({
+          index: testIndex,
+          mappings: {
+            properties: {
+              name: { type: 'text' },
+              age: { type: 'integer' },
+              '@timestamp': { type: 'date' },
```

Comment:
this field is specific to datastreams in ES. We can keep it for tests but for regular data indices it's not needed